apps/blockchain/starknet/src/tests/bridge_t.cairo

```diff
@@ -831,6 +831,102 @@ mod tests {
 
     }
 
+    #[test]
```

Comment:
Adding event checking is a good idea, but you should update `whitelist_collection_is_updated_when_collection_is_removed` instead.

Reply:
Just saw it and I've updated.
Thanks @ptisserand